stock_split_picking/models/stock_picking.py

```diff
@@ -12,9 +12,13 @@ class StockPicking(models.Model):
 
     _inherit = "stock.picking"
 
+    def _validate_split(self):
+        pass
+
     def split_process(self):
         """Use to trigger the wizard from button with correct context"""
         for picking in self:
+            self._validate_split()
 
             # Check the picking state and condition before split
             if picking.state == "draft":
```

Comment:
If the hook is meant to add more custom validations, wouldn't it make sense to include the standard ones already inside the hook?

Reply:
I agree with @LoisRForgeFlow. I would also suggest to use the word check instead of validate for the method, like _check_split_process. WDYT?

Reply:
@LoisRForgeFlow @JordiMForgeFlow Both of your feedback make sense